src/js/utils/fullscreen-api.js

```diff
@@ -1,27 +1,36 @@
 import { document } from './server-safe-globals.js';
 
 export const fullscreenApi = {
-  enter: 'requestFullscreen',
-  exit: 'exitFullscreen',
-  rootEvents: ['fullscreenchange'],
-  mediaEvents: [],
-  element: 'fullscreenElement',
-  error: 'fullscreenerror',
-  enabled: 'fullscreenEnabled',
-};
-
-if (document.fullscreenElement === undefined) {
-  fullscreenApi.enter = 'webkitRequestFullScreen';
-  fullscreenApi.exit =
-    document.webkitExitFullscreen != null
+  enter:
+    'requestFullscreen' in document
+      ? 'requestFullscreen'
+      : 'webkitRequestFullScreen' in document
```

Comment:
are these always available in the document if the target is actually the media element?

Reply:
Thanks for the catch (discussing out of band)